xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2411,6 +2474,11 @@ void CActiveAE::DeviceChange()
   m_controlPort.SendOutMessage(CActiveAEControlProtocol::DEVICECHANGE);
 }
 
+bool CActiveAE::HaveDSP()
+{
+  return m_settings.dspaddonsenabled;
```

Comment:
m_settings is not supposed to be accessed by the outside world. you could store the state in stats like CActiveAE::IsSuspended()